controllers/management.js

```diff
@@ -0,0 +1,14 @@
+const { removeHostById } = require('../services/fleet');
+
+exports.removeHost = async (req, res) => {
+    try {
+        const hostId = req.params.id;
+        const hostInfo = req.body; // Ensure the client sends the necessary host info in the request body
```

Comment:
Add input validation

Reply:
I deleted all the changes related to "remove host", and I will open a new branch for it.